backend/core/helpers.py

```
@@ -627,15 +627,20 @@ def applied_control_per_reference_control(user: User):
 ):
+    scoped_folder = (
+        Folder.objects.get(id=folder_id) if folder_id else Folder.get_root_folder()
+    )
     (
         object_ids_view,
         _,
         _,
-    ) = RoleAssignment.get_accessible_object_ids(
-        Folder.get_root_folder(), user, RiskScenario
-    )
+    ) = RoleAssignment.get_accessible_object_ids(scoped_folder, user, RiskScenario)
```

Ruff (0.8.2)

`RiskScenario` may be undefined, or defined from star imports

(F405)